packages/visual-dev/CHANGELOG.md

```diff
@@ -7,6 +7,92 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.3] - 2022-09-
```

Comment:
This feels like a bit more than a patch update, maybe it needs a minor version bump instead?